### DISCUSSION TITLE
Trouble Updating Moose

### DISCUSSION THREAD

**Discussion by Ryan7291** [idaholab/moose#28186] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Trying to update Moose by deleting the environment and reinstalling. But this results in 'Solving environment: ' running indefinitely (+6 hours). (Mac M1) Terminal below...

`(base) Ryan@Ryans-Air-4274 ~ % conda env remove -n moose'
'(base) Ryan@Ryans-Air-4274 ~ % conda create -n moose moose-dev=2024.07.15'
'Collecting package metadata (current_repodata.json): done'
'Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.'
'Collecting package metadata (repodata.json): done'
'Solving environment: \ '

**Comment by GiudGiud** [comment]
start with `conda update conda` to get the latest version of the conda solver. it s significantly faster

**Reply by Ryan7291** [reply]
Thank you, did that now it can't find plugin:

```
(base) Ryan@Ryans-Air-4274 ~ % conda env remove -n moose                 
Could not load conda plugin `conda-libmamba-solver`:
```

Much longer error message below (not shown)

**Reply by GiudGiud** [reply, marked as answer]
looks like conda is broken

I d remove it and reinstall it. we have the install instructions for conda here at the beginning
https://mooseframework.inl.gov/getting_started/installation/conda.html
